src/classes/frModel.cls

```diff
@@ -5,20 +5,29 @@ public abstract class frModel {
     public virtual void populateFromRequest(Map<String, Object> request) {
     	Map<String, Schema.SObjectField> fields = frSchemaUtil.getFields(getObject().getSObjectType().getDescribe().getName());
 
-        Map<String, frMapping__c> frNameToSfName = new Map<String, frMapping__c>();
+        //a single funraise field might flow into multiple salesforce fields
+        Map<String, List<frMapping__c>> frNameToSfName = new Map<String, List<frMapping__c>>();
```

Comment:
Hm good question.  Based on https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_set.htm it says 

> Uniqueness of all other non-primitive types is determined by comparing the objects’ fields.
So for these Custom metadata, it would never be possible for their fields to be the same because they have a unique constraint on the `name` field by default that we can't turn off, so i could make this change but I think effectively it would result in the same behavior